students/alvaro-tejedor/Ejercicios/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="style.css">
+    <title>Documento HTML</title>
+</head>
+<body>
+    <h1 id="inicio">Este es el H1 de la página</h1>
+    <img src="https://concepto.de/wp-content/uploads/2015/03/paisaje-e1549600034372.jpg" class="first-image" alt="imagen paisaje" />
+    <h2>Aquí tendríamos un H2</h2>
+    <p><b>Lorem ipsum</b> dolor sit amet consectetur, adipisicing elit. Consectetur sed esse placeat cupiditate magni 
+        voluptatibus maiores atque minima optio. <i>Libero deleniti expedita amet </i>ad ratione neque <u>aperiam corporis</u> molestiae ab.</p>
+    <ul>
+        <li><a href="https://www.google.com/" target="_blank"> Este enlace te lleva a Google </a></li>
+        <li><a href="mailto:example@email.com">Enviar email a example email</a></li>
+        <li><a href="tel:000000000">Contactar vía teléfono con servicio al cliente</a></li>
+    </ul>
+    <table border="1">
+        <tr>
+            <th>Tamaño del disco según fabricante</th>
+            <th>Tamaño real del disco en Gygabites</th>
+        </tr>   
+        <tr>
+            <td>160 GB</td>
+            <td>149 GB</td>
+        </tr>
+        <tr>
+            <td>250 GB</td>
+            <td>232 GB</td>
+        </tr>
+        <tr>
+            <td>320 GB</td>
+            <td>298 GB</td>
+        </tr>
+        <tr>
+            <td>500 GB</td>
+            <td>465 GB</td>
+        </tr>
+        <tr>
+            <td>1000 GB</td>
+            <td>931 GB</td>
+        </tr>
+        <tr>
+            <td>1500 GB</td>
+            <td>1396 GB</td>
+
+        </tr>
+        <tr>
+            <td>2000 GB</td>
+            <td>1862 GB</td>
+        </tr>
+        <tr>
+            <td>3000 GB</td>
+            <td>2793 GB</td>
+        </tr>
+    </table>
+
+    <div>
+        <form class="form">
+            <h2 class="titulo-form">Formulario</h2>
+            <input type="text" placeholder="Escribe tu nombre">
+            <input type="password" placeholder="Contraseña">
+            <input type="date">
+            <input type="email" placeholder="Escribir email">
+            <div class="input-label">
+                <label>Escoge el género</label>
```

Comment:
`label` tag must be related to an input. You can link with the input using the `for`attribute of the `label`